apis/clusters/v1beta1/cassandra_types.go

```diff
@@ -79,6 +74,47 @@ type CassandraDataCentre struct {
 	PrivateIPBroadcastForDiscovery bool `json:"privateIpBroadcastForDiscovery"`
 	ClientToClusterEncryption      bool `json:"clientToClusterEncryption"`
 	ReplicationFactor              int  `json:"replicationFactor"`
+
+	// Adds the specified version of Debezium Connector Cassandra to the Cassandra cluster
+	// +kubebuilder:validation:MaxItems=1
+	Debezium []DebeziumCassandraSpec `json:"debezium,omitempty"`
+}
+
+type DebeziumCassandraSpec struct {
+	// KafkaVPCType with only VPC_PEERED supported
+	KafkaVPCType      string `json:"kafkaVpcType"`
+	KafkaTopicPrefix  string `json:"kafkaTopicPrefix"`
+	KafkaDataCentreID string `json:"kafkaCdcId"`
+	Version           string `json:"version"`
+}
+
+func (d *CassandraDataCentre) DebeziumToInstAPI() []*models.Debezium {
+	var instDebezium []*models.Debezium
+	for _, k8sDebezium := range d.Debezium {
+		instDebezium = append(instDebezium, &models.Debezium{
+			KafkaVPCType:      k8sDebezium.KafkaVPCType,
+			KafkaTopicPrefix:  k8sDebezium.KafkaTopicPrefix,
+			KafkaDataCentreID: k8sDebezium.KafkaDataCentreID,
+			Version:           k8sDebezium.Version,
+		})
+	}
+	return instDebezium
+}
+
+func (d *CassandraDataCentre) DebeziumEquals(other *CassandraDataCentre) bool {
+	if len(d.Debezium) != len(other.Debezium) {
+		return false
+	}
+
+	for _, old := range d.Debezium {
+		for _, new := range other.Debezium {
+			if old.KafkaVPCType != new.KafkaVPCType || old.KafkaTopicPrefix != new.KafkaTopicPrefix || old.KafkaDataCentreID != new.KafkaDataCentreID || old.Version != new.Version {
+				return false
+			}
+		}
+	}
+
+	return true
```

Comment:
This struct does not contain fields that are not comparable, so you can write just `if old != new { return false }`

Reply:
Oh, yep, you're right. Fixed.